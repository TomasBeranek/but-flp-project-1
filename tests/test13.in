C,D
a
C
C->aaaaa
C->#
C->aaa
C->a
