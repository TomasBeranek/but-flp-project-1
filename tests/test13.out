0,1,2,3,4,5,6,7,8,9,10
a
2
2,6,9,10
0,a,1
1,a,4
2,a,0
2,a,7
2,a,10
4,a,5
5,a,6
7,a,8
8,a,9
